doc/developer-notes.md

```diff
@@ -461,6 +461,8 @@ General Bitcoin Core
   - *Explanation*: If the test suite is to be updated for a change, this has to
     be done first.
 
+  - ***Developers using their own Travis CI account*** to pretest their commits should work off of a recent master that is passing. This ensures that your work is based on a passing version and any fails are contained within your commit. ***You may have to restart your builds. See: [Build Timeouts](https://docs.travis-ci.com/user/customizing-the-build#build-timeouts)***
```

Comment:
Adding `***` around this is overkill, it is not significantly more important than the other recommendations. Please format this like the other items.